design/numeral/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { classNames } from '@oacore/design/lib/utils'
+
+import styles from './styles.css'
+
+const Numeral = ({
+  children,
+  className,
+  size,
+  tag: Tag = 'div',
+  ...restProps
+}) => (
+  <Tag
+    className={classNames.use(styles.container, styles[size]).join(className)}
+    {...restProps}
+  >
+    {children}
+  </Tag>
+)
+
+const Appendix = ({ children, className, tag: Tag = 'span', ...restProps }) => (
+  <Tag
+    className={classNames.use(styles.appendix).join(className)}
+    {...restProps}
+  >
+    {children}
+  </Tag>
+)
+
+const Caption = ({ children, className, tag: Tag = 'span', ...restProps }) => (
+  <Tag
+    className={classNames.use(styles.caption).join(className)}
+    {...restProps}
+  >
+    {children}
+  </Tag>
+)
+
+const Diff = ({ children, className, tag: Tag = 'span', ...restProps }) => (
+  <Tag className={classNames.use(styles.diff).join(className)} {...restProps}>
+    {children}
+  </Tag>
+)
+
+const Value = ({ children, className, tag: Tag = 'span', ...restProps }) => (
```

Comment:
I would make these all React.memo. It's a very small improvement but still

Reply:
Can you do it?

Reply:
Sorry, yes